draft-ietf-quic-recovery.md

```diff
@@ -420,8 +420,9 @@ The RECOMMENDED initial value for the packet reordering threshold
 less than 3, to keep in line with TCP {{?RFC5681}}.
 
 Some networks may exhibit higher degrees of reordering, causing a sender to
-detect spurious losses.  Implementers MAY use algorithms developed for TCP, such
-as TCP-NCR {{?RFC4653}}, to improve QUIC's reordering resilience.
+detect spurious losses.  Increasing reordering thresholds after losses
+are spuriously detected have proven useful in TCP, such as TCP-NCR
+{{?RFC4653}}, and are expected to be equally useful in QUIC.
```

Comment:
The "such as" is injected at an odd point here, which seems to break up the flow.

```suggestion
detect spurious losses.  Increasing reordering thresholds after losses, as
recommended in TCP-NCR {{?RFC4653}}, have proven to be useful in TCP and are
expected to be equally useful in QUIC.
```